website/integrations/services/engomo/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: engomo
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is engomo
+
+> engomo is an low-code app development platform to create enterprise apps for smartphones and tablets based on Android, iOS, or iPadOS.
+> -- https://engomo.com/
+>
+> This guide explains how to set up engomo to use authentik as the OAUTH provider for the application login on the smartphone/tablet and login to the admin WebGUI (composer).
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `engomo.company` is the FQDN of the Engomo install.
+-   `authentik.company` is the FQDN of the authentik install.
+-   `engomo.mapping` is the name of the Scope Mapping.
+-   `ak.cert` = The authentik self-signed certificate you use for the service provider.
+
+## authentik configuration
+
+In authentik, create a new scope mapping. To do so, log in to the Admin interface, and navigate to **Customization-->Property Mapping** and then click **Create**.
+
+-   `engomo.mapping` is the value for the Name.
+-   `profile` is the value for the Scope name.
+-   `return {"preferred_username": request.user.email,}` is the value for the Expression.
+
+Create an application and OAuth2/OpenID provider in authentik. Use the following parameters for the OAuth2/OpenID provider::
+
+Provider:
+
+-   Name: `SP-engomo`
+-   Client type: `Public`    
+-   Redirect URIs/Origins (RegEx): `https://engomo.company/auth` and `com.engomo.engomo://callback/`
+-   Signing Key: `ak.cert`
+-   Scopes: `authentik default OAuth Mapping: OpenID 'email'` and `authentik default OAuth Mapping: OpenID 'offline_access'` and `authentik default OAuth Mapping: OpenID 'openid'` and `engomo.mapping`
+-   Note the Client ID and Client Secret.
+
+> [!IMPORTANT]
```

Comment:
What do you mean? The Scopes?

We struggled in my company with a huge firewalling company that provides IDP as well (that we have at work). There we figured out that their IDP isn't using any standard scopes. You always have to add them one by one and only all 4 of them did work.

I also got in contact with the dev/support guys of engomo. They told me exactly what their application needs.
What I think about right now is that we don't need the certificate... at least I guess so. Will check and report back.

Reply:
> When looking at review comments look at the line before the comment starts. in this case it would be the IMPORTANT and that block.
> 
> I was saying that i'm not sure the block is needed as it can be guessed and the notice is not present in other integration pages.

so I comma seperated now and remove the "note line" ok?